docs/gather-job.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: insights-operator-job
+  annotations:
+    config.openshift.io/inject-proxy: insights-operator
+spec:
+  backoffLimit: 3
+  ttlSecondsAfterFinished: 600
+  template:
+    spec:
+      restartPolicy: OnFailure
+      serviceAccountName: operator
+      nodeSelector:
+        beta.kubernetes.io/os: linux
+        node-role.kubernetes.io/master: ""
+      tolerations:
+      - effect: NoSchedule
+        key: node-role.kubernetes.io/master
+        operator: Exists
+      - effect: NoExecute
+        key: node.kubernetes.io/unreachable
+        operator: Exists
+        tolerationSeconds: 900
+      - effect: NoExecute
+        key: node.kubernetes.io/not-ready
+        operator: Exists
+        tolerationSeconds: 900
+      volumes:
+      - name: snapshots
+        emptyDir: {}
+          #sizeLimit: 1Gi # bug https://bugzilla.redhat.com/show_bug.cgi?id=1713207
+      - name: trusted-ca-bundle
+        configMap:
+          name: trusted-ca-bundle
+          optional: true
+      - name: service-ca-bundle
+        configMap:
+          name: service-ca-bundle
+          optional: true
+      - name: serving-cert
+        secret:
+          secretName: openshift-insights-serving-cert
+          optional: true
+      initContainers:
+      - name: insights-operator
+        image: quay.io/openshift/origin-insights-operator:latest
```

Comment:
To test it you should replace this with an image that has the gather_job.go changes in it. (you can use mine: `quay.io/0sewa0/oneoff` or build your own :) )

Reply:
For testing it's OK but I think we should recommend users to use the same image they have in their original IO deployment (Just a note for documentation... :)) .